openglider/glider/rib/rib.py

```diff
@@ -281,9 +282,10 @@ def rib_rotation(aoa: float, arc: float, zrot: float, xrot: float=0) -> euklid.v
     return rot3 * rot2 * rot1 * rot0
 
 
-def rib_transformation(aoa: float, arc: float, zrot: float, xrot: float, scale: float, pos: euklid.vector.Vector3D) -> euklid.vector.Transformation:
+def rib_transformation(aoa: float, arc: float, zrot: float, xrot: float, scale: float, pos: euklid.vector.Vector3D, offset: float) -> euklid.vector.Transformation:
     scale_transform = euklid.vector.Transformation.scale(scale)  # type: ignore
     #scale = Scale(scale)
+    pos = pos + euklid.vector.Vector3D([0, 0, offset])
```

Comment:
eventuell in lokaler z-richtung offsetten?

also:

``` python
offset = euklid.vector.Transformation.translation(Vector3D([0,0,offset])
...
return scale_transform * offset * rot * move
```

Reply:
funktioniert irgendwie nicht ?